injector/src/test/java/edu/ucr/cs/riple/injector/AnnotationWithArgumentTest.java

```diff
@@ -51,8 +51,7 @@ public void onFieldSuppressWarningTest() {
         .expectOutput(
             "package com.edu;",
             "public class Super {",
-            "   @SuppressWarnings(\"NullAway.Init\")",
-            "   Object h = new Object();",
+            "   @SuppressWarnings(\"NullAway.Init\") Object h = new Object();",
```

Comment:
Mmm... I wonder if GJF will re-insert the new line here or not. 

Feels like we'd hope it does, since it matches how developers insert these suppressions better. Not sure if anyone minds these being in the same line, though, so I'd say this is fine for now. In the future, we might have to revert this decision and do `\n` in some cases and ` ` in others (should be one character either way so it shouldn't be too complicated to track, right?)

Reply:
Yes exactly, we can change this in future if this becomes an issue.